shared_model/validators/validators_common.cpp

```diff
@@ -5,16 +5,19 @@
 
 #include "validators/validators_common.hpp"
 
+#include <ametsuchi/impl/settings_impl.hpp>
 #include <regex>
 
 namespace shared_model {
   namespace validation {
 
     ValidatorsConfig::ValidatorsConfig(uint64_t max_batch_size,
+                                       std::shared_ptr<iroha::ametsuchi::Settings> settings,
                                        bool partial_ordered_batches_are_valid)
         : max_batch_size(max_batch_size),
-          partial_ordered_batches_are_valid(partial_ordered_batches_are_valid) {
-    }
+          partial_ordered_batches_are_valid(partial_ordered_batches_are_valid),
+          settings(settings ? std::move(settings) : std::make_shared<iroha::ametsuchi::SettingsImpl>(nullptr))
```

Comment:
I think settings should be passed explicitly. Referencing the above comments, I would pass either in-memory default storage or WSV storage when needed.